dist/TempManager.js

```diff
@@ -0,0 +1,46 @@
+class TempManager {
+    constructor() {
+        this.cityData = []
+    }
+
+    async getDataFromDB() {
+        let cities = await $.get('/cities')
+        this.cityData = cities
+    }
+
+    async getCityData(cityName) {
+            const data = await $.get(`/city/${cityName}`)
+            if (!data.name) {
+                alert("You've entered an invalid city name")
+                return
+            }
+            this.cityData.push(data)
```

Comment:
maybe use `.unshift()` so it will be pushed to the front of the array and will render at the top of the app screen